ironic-config/dnsmasq.conf.j2

```diff
@@ -12,8 +12,19 @@ log-dhcp
 dhcp-range={{ env.DHCP_RANGE }}
 
 # Disable default router(s) and DNS over provisioning network
```

Comment:
This comment should be moved

Reply:
... or even removed completely since you have another comment just below

Reply:
remove that comment.